.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,139 @@
+name: Nightly
+
+on:
+  workflow_dispatch:
+
+jobs:
+  retrieve-info:
+    runs-on: ubuntu-22.04
+    outputs:
+      artifacts-link: ${{ steps.artifacts.outputs.link }}
+      version: ${{ steps.get-version.outputs.version }}
```

Comment:
Apparently :smile: https://docs.github.com/en/actions/learn-github-actions/variables#naming-conventions-for-configuration-variables

But I will change it, so that it's clearer